src/main/kotlin/org/sert2521/infiniterecharge2020/drivetrain/Drivetrain.kt

```diff
@@ -3,29 +3,55 @@ package org.sert2521.infiniterecharge2020.drivetrain
 import com.ctre.phoenix.motorcontrol.ControlMode
 import com.kauailabs.navx.frc.AHRS
 import edu.wpi.first.wpilibj.I2C
+import edu.wpi.first.wpilibj.geometry.Rotation2d
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveOdometry
+import kotlin.math.IEEErem
 import org.sert2521.infiniterecharge2020.MotorControllers
 import org.sert2521.sertain.coroutines.RobotScope
+import org.sert2521.sertain.events.onTick
 import org.sert2521.sertain.motors.MotorController
 import org.sert2521.sertain.subsystems.Subsystem
 import org.sert2521.sertain.telemetry.Table
 import org.sert2521.sertain.telemetry.TableEntry
 import org.sert2521.sertain.telemetry.linkTableEntry
 import org.sert2521.sertain.telemetry.withTableEntry
+import org.sert2521.sertain.units.Chronic
+import org.sert2521.sertain.units.CompositeUnit
+import org.sert2521.sertain.units.Linear
+import org.sert2521.sertain.units.Meters
+import org.sert2521.sertain.units.MetricValue
+import org.sert2521.sertain.units.Per
+import org.sert2521.sertain.units.Radians
+import org.sert2521.sertain.units.Seconds
+import org.sert2521.sertain.units.convertTo
+import org.sert2521.sertain.units.div
+import org.sert2521.sertain.units.rdps
 
 class Drivetrain : Subsystem("Drivetrain", ::controlDrivetrain) {
     val table = Table(name)
 
+    init {
+        RobotScope.onTick {
+            odometry.update(
+                    Rotation2d.fromDegrees(heading),
+                    MetricValue(motorEncoder.ticks, leftDrive.position.toDouble()).convertTo(Radians).value * wheelRadius.value,
```

Comment:
Can `leftDrive` just have a property that does that conversion instead of having to do it here? So it takes care of the radians in one place instead of several?